v63004/u_inref/gtm8777.exp

```diff
@@ -0,0 +1,207 @@
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+set timeout 120
+spawn /usr/local/bin/tcsh -f
+# Change shell prompt to something other than ">" as that is a substring of the LKE prompt "LKE>"
+# and can cause incorrect match later when we wait for the shell prompt.
+expect -exact ">"
+# Note: Changing the shell prompt to SHELL might seem easily achieved as follows.
+#	send -- "set prompt=SHELL\r"
+#	expect -exact "SHELL"
+# But that will not work because it is possible the "expect" matches the SHELL from the "set prompt=SHELL" input
+# instead of from the SHELL prompt later spewed out by the shell. To avoid this, we first store the "SHELL" string
+# in a shell variable and use that variable in another line to set the prompt.
+send -- "set shellprompt=SHELL\r"
+expect -exact ">"
+send -- "set prompt=\$shellprompt\r"
+expect -exact "SHELL"
+
+expect_after {
+	timeout { timeout_procedure }
+}
+
+proc timeout_procedure { } {
+	puts "timeout occurred"
+	exit -1
+}
+
+#Start mumps -direct
+send -- "\$gtm_dist/mumps -direct\r"
+expect -exact "YDB>"
+
+#Start GCE prompt
+puts "QUIET^%GCE TEST\n"
+puts "--------------------------------------------------------------\n"
+send -- "do QUIET^%GCE\r"
+expect -exact "Global Change Every occurrence"
+expect -exact "Global ^"
+send -- "string1\r"
```

Comment:
Can you do a ZWRITE ^string1 before and after the GCE call to confirm GCE did its job. Same case with ZWRITE ^string4 in next test. And with "cat test1.m" and 'cat test2.m" in the RCE tests.